docs/use-cases/greenfield-native.md

```diff
@@ -0,0 +1,113 @@
+enable users to sign in using their existing social media accounts such as Google, Facebook, and Microsoft. Experience with their
+existing browser-based app has convinced ACME that social sign-in leads to much better conversion rates.
+
+## Implementing authentication for a native app
+
+They decided to implement the following phased plan:
+
+- [Start with a POC](#start-with-a-poc)
+- [Elaborate and customize the app](#elaborate-and-customize-the-app)
+- [Go to production](#go-to-production)
+
+### Start with a POC
+
+1. The app developer created a new project in Ory Console, choosing the US hosting location.
+
+2. The app developer then quickly [installed the Ory CLI](https://www.ory.sh/docs/guides/cli/installation) to work with the new
+   project. At this point, the app developer's environment was ready to start work on integrating the app with Ory Network.
+
+3. The developer decided to build the POC for the mobile app using the [React Native](https://reactnative.dev/) framework, as this
+   provides a platform-neutral framework for building mobile apps on Android and iOS. The developer soon located Ory's
+   [React Native quickstart](https://www.ory.sh/docs/getting-started/integrate-auth/react-native), which could serve as a model
+   for the POC.
+
+4. The initial POC planned to support just three flows: account registration, login, and logout. Because the React Native
+   quickstart already provides sample implementations of these authentication flows, this phase of the coding was completed
+   quickly.
+
+5. The app developer completed the POC ahead of schedule and successfully demonstrated the authentication flows to project
+   managers at ACME Corporation.
+
+### Elaborate and customize the app
+
+At this point, the developer team at ACME Corporation were ready to extend the functionality of the app and customize the features
+of Ory Network.
+
+1. Adding the self-service account settings flow was a quick win for the developer team, because the React Native quickstart
+   already provides a sample implementation for this.
+
+2. The developer team also added the account verification flow to the app.
+
+3. As ACME needed to add custom fields to the account registration form, the developer team decided to
+   [customize the identity schema](https://www.ory.sh/docs/kratos/manage-identities/customize-identity-schema) for their project.
+
+4. To support the account verification flow, they needed to integrate the Ory project with their email server. At this point, they
+   called in their system administrator to perform the following tasks on Ory Console:
+
+   - Customize the email template for the account verification flow.
+   - Integrate the Ory project with their corporate SMTP email server.
+
+5. ACME Corporation decided to enable social sign-in for the initial rollout of their app, as they expect it will accelerate user
+   registrations.
```

Comment:
This thing is repeated quite for the third (?) time at this point without giving more details.

Reply:
Ok, I left out the mention of accelerated user registrations in this step.